README.md

```diff
@@ -218,6 +218,79 @@ Once you collect all information and create a case study, open a pull request. I
 
 A case study becomes publicly available right after merging and rebuilding the website.
 
+# 🌐 Adding Short URLs to the AsyncAPI Website
```

Comment:
Never use emojis in headings, they break anchor urls.

Please [check the reply in another PR with the same issue](https://github.com/asyncapi/website/pull/4457#discussion_r2445687267)